app/assets/javascripts/application.js

```diff
@@ -131,6 +131,26 @@ function sync() {
   $("a.sync").click();
 }
 
+function scrollToCursor() {
+  table_offset = $('.table-notifications').position().top;
+  cursor_offset = $('td.current').offset().top;
+  cursor_relative_offset = $('td.current').position().top;
+  cursor_height = $('td.current').height();
+  menu_height = $("#octobox-menu").height();
+  scroll_top = $(document).scrollTop();
+  window_height = $(window).height();
```

Comment:
Yeah so the markup would have things like `js-table-notifications` `js-current` and `js-octobox-menu`